src/FixedDataTableRow.js

```diff
@@ -452,22 +457,21 @@ class FixedDataTableRow extends React.Component {
       width: this.props.width,
       height: this.props.height,
       zIndex: (this.props.zIndex ? this.props.zIndex : 0),
+      display: (this.props.visible ? 'block' : 'none'),
     };
     FixedDataTableTranslateDOMPosition(style, 0, this.props.offsetTop, this._initialRender);
 
+    const { offsetTop, zIndex, visible, ...rowProps } = this.props;
```

Comment:
Cleanup for https://github.com/schrodinger/fixed-data-table-2/pull/415#discussion_r270631650